integration/test/metric_value_benchmark/agent_configs/base_linux_config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "agent": {
+    "metrics_collection_interval": 60,
+    "run_as_user": "root",
+    "debug": true,
+    "logfile": ""
+  },
+  "metrics": {
+    "metrics_collected": {
+      "mem": {
+        "measurement": [
+          "mem_used_percent"
+        ]
+      },
+      "disk": {
+        "measurement": [
+          "used_percent"
+        ],
+        "resources": [
+          "*"
+        ]
+      }
+    },
+    "append_dimensions": {
+      "ImageId": "${aws:ImageId}",
+      "InstanceId": "${aws:InstanceId}",
+      "InstanceType": "${aws:InstanceType}",
+      "AutoScalingGroupName": "${aws:AutoScalingGroupName}"
```

Comment:
Are all these needed? Is this the minimum required by agent if customers want to run the agent? If this isn't really about our product spec and just for the dummy test, we can remove dummy test. I had added that just to give an example for others on how future tests can be added

Reply:
I put this in here because I didn't want to reuse the CPU or mem test config for the "dummy" test. This is taken directly from our existing default Linux configuration.

Reply:
🤦 omg I just realized I didn't delete the dummy test (I didn't commit and push). I'll make a followup PR to just remove the dummy test.